stdlib/pstats.pyi

```diff
@@ -51,9 +51,9 @@ if sys.version_info >= (3, 9):
         tottime: float
         percall_tottime: float
         cumtime: float
-        percall_cumtime: float
-        file_name: str
-        line_number: int
+        percall_cumtime: float = ...
+        file_name: str = ...
+        line_number: int = ...
```

Comment:
I'd be more interested in an actual function call on the r.h.s., e.g.

```suggestion
        percall_cumtime: float = field(init=False, default_factory=list)
        file_name: str = Field(default_factory=Incomplete)
        line_number: int = ...
```

with `field` and `Field` imported from dataclasses at the top of the file

Reply:
Working on it 😅 Seems ruff didn't like it and reformatted it during the auto pre-commit run.

Reply:
Aah I see! This kind of thing violates some assumptions made by https://docs.astral.sh/ruff/rules/typed-argument-default-in-stub/ (and the same rule in the flake8-pyi plugin as well), I suppose